### PR TITLE
Added entry for GDG Liverpool DevFest

### PR DESCRIPTION
**Pull request by blundell**

### CODE REVIEW

File: _conferences/gdg_liverpool_devfest_2016.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: "GDG Devfest"
+website: https://docs.google.com/forms/d/e/1FAIpQLSfT8oMoJHyOjNGyDVE15O-TEw5I8ABeVNeJRywj7OXlBHFdqg
```
Comment (ViliusKraujutis) [inline _conferences/gdg_liverpool_devfest_2016.md:3]: @blundell, looks like this file is not available.

image

File: _conferences/gdg_liverpool_devfest_2016.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+location: Liverpool, UK
+
+date_start: 2016-11-25
+date_end:   2016-11-25
+
+cfp_start: 2016-09-01  
+cfp_end:   2016-11-01  
+cfp_site: https://docs.google.com/forms/d/e/1FAIpQLSfT8oMoJHyOjNGyDVE15O-TEw5I8ABeVNeJRywj7OXlBHFdqg
```
Comment (ViliusKraujutis) [inline _conferences/gdg_liverpool_devfest_2016.md:11]: ditto

File: _conferences/gdg_liverpool_devfest_2016.md (hunk 1 above)
Comment (blundell) [reply _conferences/gdg_liverpool_devfest_2016.md:3]: ah so strange its /viewform at the end , will update

Comment (ViliusKraujutis) [reply _conferences/gdg_liverpool_devfest_2016.md:3]: `/edit` is added automatically.

Comment (ViliusKraujutis) [reply _conferences/gdg_liverpool_devfest_2016.md:3]: I'd recommend you checking that link in an anonymous session in your Internet browser.

Comment (blundell) [reply _conferences/gdg_liverpool_devfest_2016.md:3]: updated https://github.com/AndroidStudyGroup/conferences/pull/123

File: _conferences/gdg_liverpool_devfest_2016.md (hunk 2 above)
Comment (blundell) [reply _conferences/gdg_liverpool_devfest_2016.md:11]: https://github.com/AndroidStudyGroup/conferences/pull/119#discussion_r80419995

**Pull request merged**